library_generation/utilities.sh

```diff
@@ -174,17 +165,20 @@ download_from() {
   curl -LJ -o "${save_as}" --fail -m 30 --retry 2 "$url" || download_fail "${save_as}" "${repo}"
 }
 
+# copies the specified file in $1 to $2
+# will return "false" if the copy was successful, otherwise true (to indicate
```

Comment:
It maybe unintuitive to indicate a successful copy as `false`, though it's a minor issue.

Reply:
I inverted the logic and changed variable names accordingly